src/System.Net.Sockets/tests/FunctionalTests/SocketOptionNameTest.cs

```diff
@@ -87,6 +87,11 @@ public void MulticastOption_CreateSocketSetGetOption_GroupAndInterfaceIndex_SetS
         [Fact]
         public async Task MulticastInterface_Set_AnyInterface_Succeeds()
         {
+             if (PlatformDetection.IsFedora || PlatformDetection.IsRedHatFamily7)
```

Comment:
Isn't there a way to disable the test via `ActiveIssue` for just specific OSs?

Reply:
I'm not sure. This is how it was done before. #25552